crates/turborepo/build.rs

```diff
@@ -90,21 +26,38 @@ fn build_debug_libturbo() -> PathBuf {
         for ext in ["pdb", "exe", "d", "lib"].iter() {
```

Comment:
This probably isn't necessary anymore

Reply:
Not a windows expert here. @chris-olszewski confirmed?

Reply:
I believe that's correct.